apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/internal/CacheMapBuilder.kt

```diff
@@ -0,0 +1,116 @@
+package com.apollographql.apollo.cache.normalized.internal
```

Comment:
What do you think of moving this class into sub package `com.apollographql.apollo.cache.normalized.internal.batching/batch/batchreader`?

Reply:
Sounds good to me!